doc/source/examples/sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer.rst

```diff
@@ -0,0 +1,297 @@
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.01
+      end
+    end
+
+
+Mesh and Mesh Adaptation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The mesh is a simple hyper rectangle, large enough to encompass the mixer with its casing and long enough to establish the flow profile upstream and downstream.
+
+
+.. code-block:: text
+
+    subsection mesh
+      set type      = dealii
+      set grid type = subdivided_hyper_rectangle
+
+      # Grid to use when solving the flow in the long version of the mixer
+      set grid arguments = 6,1,1: -150,-25,-25: 150,25,25: true
+
+      set initial refinement = 3
+    end
+
+Mesh adaptation ``type`` is set to ``kelly``, to allow adaptive refinement at the solid surface. This is necessary for simulations of this type because of the prohibitive mesh size required when only uniform refinement is used. Setting ``max refinement level = 5`` allows for two levels of adaptive refinement from the uniform ``initial refinement = 3`` defined in the ``mesh`` section. The ``frequency = 10000`` to ensure that no refinement occurs between time steps, as they are not necessary here.
+
+.. code-block:: text
+
+    subsection mesh adaptation
+      set type                 = kelly
+      set fraction type        = number
+      set max number elements  = 2000000
+      set max refinement level = 5
+      set min refinement level = 0
+      set frequency            = 100000
+    end
+
+
+
+Definition of the Shape
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The section defining each parameter for the particles has certain requirements:
+
+#. ``length ratio`` defines the length used to apply the immersed boundaries through interpolation. We choose ``4`` as a compromise between a low value, better for the linear solver, and a high value, better for mass preservation. Mass preservation can also be increased using a finer grid.
+#. ``refine mesh inside radius factor`` and ``refine mesh outside radius factor`` are both set to ``1``, which activates minimal crown refinement mode.
+#. ``type = composite`` and ``shape arguments = mixer_long.composite`` allow to refer the defined complex shape. This requires that the ``RBF_helix.output`` is located in the same directory as the parameter file.
+
+.. code-block:: text
+
+    subsection particles
+      set assemble Navier-Stokes inside particles = false
+      set number of particles                     = 1
+
+      subsection extrapolation function
+        set length ratio  = 4
+        set stencil order = 1
+      end
+
+      subsection local mesh refinement
+        set initial refinement                = 4
+        set refine mesh inside radius factor  = 1
+        set refine mesh outside radius factor = 1
+        set refinement zone extrapolation     = false
+      end
+
+      subsection particle info 0
+        set type            = composite
+        set shape arguments = mixer_long.composite
+      end
+    end
```

Comment:
I would maybe move the code-block before the list of requirements, and mention the associated subsection for each element of the list.